pkgs/development/python-modules/numba/default.nix

```
@@ -26,8 +23,7 @@ buildPythonPackage rec {
 
   NIX_CFLAGS_COMPILE = lib.optionalString stdenv.isDarwin "-I${libcxx}/include/c++/v1";
 
-  propagatedBuildInputs = [numpy llvmlite]
-    ++ lib.optionals isPy27 [ funcsigs singledispatch];
+  propagatedBuildInputs = [numpy llvmlite setuptools];
```

since we are editting the file anyway.
```suggestion
  propagatedBuildInputs = [ numpy llvmlite setuptools ];
```